File: BaseLibrary/src/main/java/com/gcm/heruijun/base/ui/adapter/BaseAdapter.java
package com.gcm.heruijun.base.ui.adapter;

import java.util.List;

/**
 * Created by heruijun on 2017/12/11.
 */
public interface BaseAdapter<T> {

    T getItem(int position);

    int getItemViewType(int position);

    void add(T item);

}

File: BaseLibrary/src/main/java/com/gcm/heruijun/base/ui/adapter/BaseAdapter.kt
package com.gcm.heruijun.base.ui.adapter

/**
 * Created by heruijun on 2017/12/11.
 */
interface BaseAdapter<T> {

    fun getItem(position: Int): T

    fun getItemViewType(position: Int): Int

    fun add(item: T)

}